gmail-to-salesforce-lead/README.md

```diff
@@ -0,0 +1,80 @@
+# Example to create leads obtained via email on Salesforce using the OpenAI API.
+
+## Use case
+The following sample demonstrates a scenario in which customer leads obtained through email are automatically pushed to Salesforce. The required details for the lead (name, company, designation etc.) are inferred using the content of the email and the OpenAI chat API.
```

Comment:
```suggestion
The following sample demonstrates a scenario in which customer leads obtained through email are automatically pushed to Salesforce. The required details for the lead (name, company, designation, etc.) are inferred using the content of the email and the OpenAI chat API.
```

Reply:
Fixed in [a718fba](https://github.com/ballerina-guides/integration-samples/pull/32/commits/a718fba876d7ed04e212bd0cd384c113178380cb)